src/Illuminate/Http/Request.php

```diff
@@ -314,7 +314,7 @@ public function replace(array $input)
      *
      * @param  string  $key
      * @param  mixed   $default
-     * @return mixed
+     * @return \Symfony\Component\HttpFoundation\ParameterBag|mixed
```

Comment:
It doesn't make sense to perform set union with the universe.

Reply:
It's documentation. Read it as: This method returns ParameterBag or something else (depending on the input arguments). I find it very helpful and it enables PhpStorm code completion on the returned ParameterBag object.